### ISSUE TITLE
Add project vertical information to project cards: Vote / Representation

### ISSUE THREAD

**Issue by tarang100** [hackforla/website#908]
### Overview
We need to add the vertical information to each project for Vote / Representation so that when we launch the Project Verticals page it will automatically include each project.

### Action Items
Add the Field and Value
```
vertical: 'Vote / Representation'
```
to the following projects mark down files
- [x] BallotNav
- [x] UnDebate
- [x] HelloGov

### Resources/Instructions
Project files are located at [/gh-pages/_projects](https://github.com/hackforla/website/tree/gh-pages/_projects)
Example of where to put the vertical in the [project card template](https://github.com/hackforla/website/wiki/Template-of-a-project.md-file)

**Comment by pawan92** [comment]
@rh5140 Please put an update on your issue
1. Progress:
2. Blockers:
3. Availability:
4. ETA

**Comment by rh5140** [comment]
Update: Sorry, I didn't know there was an update format.

1. I finished but didn't actually complete the process of making a pull request on the website when I made the commit ;________; I opened the link to the website and called it a day before clicking "New Pull Request" lmao
2. Now I don't know how to make the pull request based on a past commit. I did some googling and tried checking the git log and using cherry-pick, but I get the error "The previous cherry-pick is now empty, possibly due to conflict resolution." I don't know how to fix this since locally, I already have the change, but it is not on either remote branch.
3. I need to finish some homework today ;_______________;
4. Depends on when I find an answer ;_; I am very sorry I made this simple issue not-so-simple
